*** ./smali/android/app/SystemServiceRegistry.smali	2017-02-13 00:07:12.331427371 +0900
--- /home/fullgreen/build/nougat/patchrom/c1skt/temp/new_smali/framework.jar.out/./smali/android/app/SystemServiceRegistry.smali	2017-02-13 00:05:04.593992930 +0900
***************
*** 750,755 ****
--- 750,757 ----
  
      invoke-static {v0, v1, v2}, Landroid/app/SystemServiceRegistry;->registerService(Ljava/lang/String;Ljava/lang/Class;Landroid/app/SystemServiceRegistry$ServiceFetcher;)V
  
+     invoke-static {}, Landroid/app/ContextImplInjector;->registerMiuiServices()V
+ 
      return-void
  .end method
  
